sdk/core/core/src/az_log.c

```diff
@@ -176,6 +176,28 @@ static az_result _az_log_http_response_msg(
   return _az_log_http_request_msg(log_msg_bldr, hrb, 1);
 }
 
+static az_result _az_log_http_retry_msg(
+    az_span_builder * const log_msg_bldr,
+    int16_t const attempt,
+    int32_t const delay_msec) {
+  AZ_RETURN_IF_FAILED(az_span_builder_append(log_msg_bldr, AZ_STR("HTTP Retry attempt")));
+
+  if (attempt >= 0) {
+    AZ_RETURN_IF_FAILED(az_span_builder_append(log_msg_bldr, AZ_STR(" #")));
+    AZ_RETURN_IF_FAILED(az_span_builder_append_uint64(log_msg_bldr, (uint64_t)attempt));
+  }
+
+  AZ_RETURN_IF_FAILED(az_span_builder_append(log_msg_bldr, AZ_STR(" will be made")));
+
+  if (delay_msec >= 0) {
+    AZ_RETURN_IF_FAILED(az_span_builder_append(log_msg_bldr, AZ_STR(" in ")));
+    AZ_RETURN_IF_FAILED(az_span_builder_append_uint64(log_msg_bldr, (uint64_t)delay_msec));
+    AZ_RETURN_IF_FAILED(az_span_builder_append(log_msg_bldr, AZ_STR("ms")));
+  }
+
+  return az_span_builder_append_byte(log_msg_bldr, '.');
```

Comment:
Remove this and add the '.' to the previous 'ms' string.

Reply:
The idea was to print "HTTP Retry attempt( #3) will be made( in 3000ms).", so that we raher won't print non-positive values, because the behavior is unknown in this case, so we don't tell what we don't know. Maybe it is better to simplify.